### ISSUE TITLE
Trained model gets low accuracy on VQA server

### ISSUE THREAD

**Issue by idansc** [VT-vision-lab/VQA_LSTM_CNN#28]
Hello, 

I trained the model as you described for 150K epochs, for some reason I'm getting only 49% accuracy, while using you pretrained parameters gives 58% as you mentioned. Does anyone have an idea what could cause such drop in performance? 
Thanks!

**Comment by idansc** [comment]
Problem was with the vgg preprocess images

**Comment by xhzhao** [comment]
@idansc Hi idansc, i got the similar problem, i got 24.32% overall accuracy with the pretrained model, and i trained the model for 150K epochs, only got 41.86% overall accuracy. If this is the problem with the vgg preprocess images, how can i fix this problem?
